tests/test_sageintacct/test_tasks.py

```
@@ -1838,3 +1841,121 @@ def test_mark_paid_on_fyle(db, mocker):
+
+
+def test_search_and_upsert_vendors_personal(db, mocker):
+    """
+    Test search_and_upsert_vendors personal
+    """
+    workspace_id = 1
+    configuration = Configuration.objects.filter(workspace_id=workspace_id).first()
+    expense_group = ExpenseGroup.objects.filter(workspace_id=workspace_id).first()
+
+    configuration.corporate_credit_card_expenses_object = 'JOURNAL_ENTRY'
+    configuration.employee_field_mapping == 'VENDOR'
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)